Classes/Issues/IssuesViewController+UpdateIssueTitle.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  IssuesViewController+UpdateTitle.swift
+//  Freetime
+//
+//  Created by B_Litwin on 10/23/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+extension IssuesViewController {
+    
+    static func updateBookmarkTitle(
+        newTitle: String,
+        bookmark: Bookmark,
+        bookmarkStore: BookmarkStore
+        )
+    {
+
+        let newBookmark = Bookmark(
+            type: bookmark.type,
+            name: bookmark.name,
+            owner: bookmark.owner,
+            number: bookmark.number,
+            title: newTitle,
+            defaultBranch: bookmark.defaultBranch
+        )
+        
+        if let index = bookmarkStore.values.index(of: bookmark) {
+            bookmarkStore.values[index] = newBookmark
+            bookmarkStore.save()
+        }
```

Comment:
Oof, we need a better way to do this... shouldn't have to create whole model then lookup based on index... `BookmarkStore` needs better APIs for finding, updating, and replacing stuff...

I guess we can tackle that later, but this should be the responsibility of the store.

Also `var values: [Bookmark]` is a publicly mutable array... 😬 this could use some cleanup....